### PR TITLE
update

### PR DESCRIPTION
**Pull request by afsaneh-rezaeinejad-deriv**
`c_setupTradeAccount` is utilized across various tests to add a trading account if it is absent. Currently, it consistently fails when attempting to add an existing trading account. The task is to modify this function to either return the status or appropriately fail the test according to the specific test requirements.
CU: https://app.clickup.com/t/20696747/WALL-4460

### CODE REVIEW

File: cypress/support/commands/wallets.js [hunk 1]
```diff
@@ -104,7 +104,11 @@ Cypress.Commands.add('c_setupTradeAccount', (wallet) => {
           .findByText("Trader's Hub")
           .should('be.visible')
       } else {
-        cy.fail('Trading account already added')
+        if (`${action}` == 'Fail') {
```
Comment (abuzar-deriv) [inline cypress/support/commands/wallets.js:107]:
```suggestion
        if (action == 'Fail') {
```

**Review by abuzar-deriv — CHANGES_REQUESTED** [review]

**Review by abuzar-deriv — APPROVED** [review]
LGTM

File: cypress/support/commands/wallets.js [hunk 2]
```diff
@@ -87,7 +87,7 @@ Cypress.Commands.add('c_checkForBanner', () => {
   cy.findByText('Enjoy seamless transactions').should('not.exist')
 })
 
-Cypress.Commands.add('c_setupTradeAccount', (wallet) => {
+Cypress.Commands.add('c_setupTradeAccount', (wallet, action = 'Fail') => {
```
Comment (meeyun-deriv) [inline cypress/support/commands/wallets.js:90]: `action = 'Fail'` parameter name does not match its purposes. Recommend to update the parameter name. 

Just a suggestion - `Cypress.Commands.add('c_setupTradeAccount', (wallet, mustCreateNewAccount = true) => {`

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**